src/config.h

```diff
@@ -60,6 +60,9 @@ typedef struct {
   size_t gcScanSize;
 
   size_t minPhoneticTermLen;
+
+  // Chained configuration data
+  void *chainedConfig;
```

Comment:
maybe we should give a ctx for each RSConfigOptions and not only one, but we should keep it for the future ..

Reply:
Yeah, I was thinking about that, but I was also hoping to have the options not be connected to their config values.. I'm not sure why that's important thought. I'll actually merge the two right now. And also make them use ArgCursor